doc/help/commands.asciidoc

```diff
@@ -285,8 +292,7 @@ Set all settings back to their default.
 
 [[config-cycle]]
 === config-cycle
-Syntax: +:config-cycle [*--pattern* 'pattern'] [*--temp*] [*--print*]
-             'option' ['values' ['values' ...]]+
+Syntax: +:config-cycle [*--pattern* 'pattern'] [*--temp*] [*--print*] 'option' ['values' ['values' ...]]+
```

Comment:
I think this should be fixed in master - could you merge the latest `master` into your branch and regenerate?